config/galaxy.yml.sample

```diff
@@ -693,11 +693,10 @@ galaxy:
   # URL (with schema http/https) of the Galaxy instance as accessible
   # within your local network - if specified used as a default by pulsar
   # file staging and Jupyter Docker container for communicating back
-  # with Galaxy via the API.  If you are attempting to setup GIEs on Mac
-  # OS X with Docker for Mac - this should likely be the IP address of
-  # your machine on the virtualbox network (vboxnet0) setup for the
-  # Docker host VM. This can found by running ifconfig and using the IP
-  # address of the network vboxnet0.
+  # with Galaxy via the API.  If you are attempting to set up GIEs on
+  # Mac OS X with Docker Desktop for Mac this should be
```

Comment:
I'd maybe change this to:
```
and your Galaxy instance runs on port 8080 this should be
```
I guess it is obvious to us, but maybe that'll help some folks ?

Reply:
@mvdbeek Sure thing, good idea.